### PR TITLE
chore(chaos): complete job with fail message

### PR DESCRIPTION
**Pull request by Zelldon**
Build correct testResult json with chaos experiment name as content

closes https://github.com/zeebe-io/zeebe-cluster-testbench/issues/88

### CODE REVIEW

File: core/chaos-workers/handleJobTest.sh [hunk 1]
```diff
@@ -102,3 +102,57 @@ failFunction() {
...
+  # then
+  echo "$failureMsg"
+  echo "expected: $expected"
+  [ "$failureMsg" == "$expected" ]
+}
+
+
+@test "create failure message with multiple fail messages" {
+  # given
+  expected="$(jq -n '{testResult: "FAILED", failureMessages: ["2", "hallo"], failureCount: 1, metaData: {}}')"
```
Comment (pihme) [inline core/chaos-workers/handleJobTest.sh:148]: In this case the optimal result would be `failureCount=2`

**Review by pihme — APPROVED** [review]
Looks good to me. Just one comment on the call with two parameters, which you currently don't use anyways. So feel free to ignore it.

File: core/chaos-workers/handleJobTest.sh (hunk 1 above)
Comment (Zelldon) [reply core/chaos-workers/handleJobTest.sh:148]: 2 because I have two messages ? Was not sure how this field is used.

Comment (pihme) [reply core/chaos-workers/handleJobTest.sh:148]: That's more or less the idea yes. But I agree that one could also have one failure and several messages for it.

**Pull request merged**